README.md

```diff
@@ -1,2 +1,42 @@
-# Mic-Hello-Microverse
-A project to showcase professionalism in software development using version control
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [Key Features](#key-features)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
```

Comment:
## Required Changes ♻️
- [ ] You have done great so far, but there are still some changes you need to make before moving to the next milestone. :+1: 
- [ ] I can see you use the [Microverse readme template](https://github.com/microverseinc/readme-template), :+1: but as required by Microverse, you do not need to remove any section from the readme (except the FAQ), all you are required to do is to remove the placeholder comments and then customize each section to match your project. This is an example of how a [professional-looking readme should be ](https://github.com/V-Blaze/news-hub/blob/dev/README.md). I'll suggest you look into this. :smile: :wave: